src/dosbox.cpp

```diff
@@ -624,20 +619,42 @@ void DOSBOX_Init(void) {
 	};
 	pstring->Set_values(midi_devices);
 	pstring->Set_help(
-	        "Device that will receive the MIDI data from emulated MPU-401.\n"
+	        "Device that will receive the MIDI data (from the emulated MIDI\n"
+	        "interface - MPU-401).\n"
 #if C_FLUIDSYNTH
 	        "Use 'fluidsynth' to select built-in software synthesiser,\n"
 	        "see the fluidsynth section for detailed configuration.\n"
 #endif
-	        "Use 'auto' to pick the first working device.");
-
-	Pstring = secprop->Add_string("midiconfig",Property::Changeable::WhenIdle,"");
-	Pstring->Set_help("Special configuration options for the device driver. This is usually the id or part of the name of the device you want to use\n"
-	                  "(find the id/name with mixer/listmidi).\n"
-	                  "Or in the case of coreaudio, you can specify a soundfont here.\n"
-	                  "When using a Roland MT-32 rev. 0 as midi output device, some games may require a delay in order to prevent 'buffer overflow' issues.\n"
-	                  "In that case, add 'delaysysex', for example: midiconfig=2 delaysysex\n"
-	                  "See the README/Manual for more details.");
+	        "Use 'auto' to pick the first working external device.");
+
+	pstring = secprop->Add_string("midiconfig", when_idle, "");
+	pstring->Set_help(
+	        "Special configuration options for the device driver.\n"
+	        "This is usually the id or part of the name of the device you\n"
+	        "want to use (find the id/name with DOS command 'mixer /listmidi').\n"
+#if C_FLUIDSYNTH
+	        "- When using built-in FluidSynth, this option has no effect.\n"
+	        "  Use section 'fluidsynth' below instead.\n"
```

Comment:
When using the built-in FluidSynth (mididevice = fluidsynth), this option has no effect. Instead, configure FluidSynth using the dedicated [fluidsynth] section below.